partner_interest_group/models/res_partner_interest_group.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResPartnerInterestGroup(models.Model):
+    _name = "res.partner.interest.group"
+    _description = "Configurable Interest Group for Partners"
+
+    name = fields.Char(string="Interest Group")
+    active = fields.Boolean(default=True)
+    res_partner_id = fields.Many2many("res.partner")
```

Comment:
I forgot to mention this in last review, if you have time to change it, it's nice. If not I can live with a `res_partner_id` field.
```suggestion
    partner_id = fields.Many2many("res.partner")
```

Reply:
fixed